pkgs/development/python-modules/pcre2-py/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  build,
+  bzip2,
+  cmake,
+  cython,
+  editline,
+  gitpython,
+  pytestCheckHook,
+  python3Packages,
+  buildPythonPackage,
+  fetchFromGitHub,
+  haskellPackages,
+  lib,
+  libedit,
+  libz,
+  pcre2,
+  scikit-build,
+  setuptools,
+  twine,
+  readline,
+  requests,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "pcre2-py";
+  version = "0.4.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "grtetrault";
+    repo = "pcre2.py";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-mivw+IITO6GTj3OnoE+nb4K2nZQRWk8OWWXIGnIYn/o=";
+    fetchSubmodules = true;
```

Comment:
Don't we want to use our pcre2?

Reply:
for this it needs python wrapped pcre2 https://github.com/ComplianceAsCode/content

Reply:
I can't follow what the link is about you posted.

I am asking if we should replace the libpcre2 directory here https://github.com/grtetrault/pcre2.py/tree/main/src with the one from pkgs.pcre2.

Reply:
Yes

Reply:
after patching out already available pcre2 package. Packaging is eperiencing error undefined symbol: pcre2_compile_8 @SuperSandro2000 and @wegank

Reply:
Then maybe it doesn't work. Hence I was asking if we want to use it or not.

Reply:
Yes we want to use it.